.devcontainer/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM renovate/buildpack:2
```

```suggestion
FROM renovate/buildpack:2-focal
```
Use ubuntu focal to have newer python.